src/rtk/features/delegableApi/delegableApiSlice.ts

```diff
@@ -47,24 +69,82 @@ const delegableApiSlice = createSlice({
   name: 'delegableApi',
   initialState,
   reducers: {
-    softAdd: (state, action) => {
+    softAdd: (state: SliceState, action: any) => {
```

Comment:
No need to have specify any as type
```suggestion
    softAdd: (state: SliceState, action) => {
```

Reply:
I agree, but my eslint gives me an error if I don't specify the types 😅

Reply:
We can have a look on it on the office when you have time, to try to to fix it :)